### ISSUE TITLE
Checkpoint not loading

### ISSUE THREAD

**Issue by ted-instabase** [namedysx/CRAFT-tensorflow#2]
I'm trying to run with the associated Google Drive checkpoint and getting this error:

```
NotFoundError (see above for traceback): Key vgg_16/Repeat/convolution_1/BatchNorm/beta not found in checkpoint
	 [[Node: save/RestoreV2_79 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_79/tensor_names, save/RestoreV2_79/shape_and_slices)]]
```

My modification was:
```
if __name__ == '__main__':
    # train(False)
    test('/Users/eob/code/CRAFT-tensorflow/ckpt/CRAFT_262000.ckpt', '/Users/eob/code/CRAFT-tensorflow/image/image/t.jpg')
```

**Comment by namedysx** [comment]
> I'm trying to run with the associated Google Drive checkpoint and getting this error:
> 
> ```
> NotFoundError (see above for traceback): Key vgg_16/Repeat/convolution_1/BatchNorm/beta not found in checkpoint
> 	 [[Node: save/RestoreV2_79 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_79/tensor_names, save/RestoreV2_79/shape_and_slices)]]
> ```
> 
> My modification was:
> 
> ```
> if __name__ == '__main__':
>     # train(False)
>     test('/Users/eob/code/CRAFT-tensorflow/ckpt/CRAFT_262000.ckpt', '/Users/eob/code/CRAFT-tensorflow/image/image/t.jpg')
> ```

try to add 
`tf.reset_default_graph()`
to

```
def test(ckpt_path, img_path, result_path='/home/user4/ysx/CRAFT/result/'):
      tf.reset_default_graph()
```

**Comment by lostemples** [comment]
same error.
NotFoundError (see above for traceback): Key vgg_16/Repeat/convolution_1/BatchNorm/beta not found in checkpoint.
when i do test('./ckpt/CRAFT_262000.ckpt', './my.jpg').
seems like error in restore.restore(sess, ckpt_path)  in craft.py test function.

**Comment by lostemples** [comment]
checked the ckpt file. 
tensor_name in ckpt is "convolution2d_1" not "convolution_1"

**Comment by anhquyetnguyen** [comment]
using tensorflow 1.14.0 work for me
